core/src/test/java/org/elasticsearch/search/aggregations/pipeline/InternalSimpleValueTests.java

```diff
@@ -55,4 +55,16 @@ protected void assertReduced(InternalSimpleValue reduced, List<InternalSimpleVal
         return InternalSimpleValue::new;
     }
 
+    @Override
+    protected void assertFromXContent(InternalSimpleValue simpleValue, ParsedAggregation parsedAggregation) {
+        ParsedSimpleValue parsed = ((ParsedSimpleValue) parsedAggregation);
+        if (Double.isInfinite(simpleValue.getValue()) == false && Double.isNaN(simpleValue.getValue()) == false) {
+            assertEquals(simpleValue.getValue(), parsed.value(), Double.MIN_VALUE);
```

Comment:
I think Luca asked this question already but I didn't find the response... why Double.MIN_VALUE?